src/App.js

```diff
@@ -41,16 +44,25 @@ function App() {
   };
 
   return (
-    <>
+    <BrowserRouter>
+    <Routes>
+      <Route path={"/"}
+      element={
+       <>
       <h1>TO-DO List</h1>
       <AddTodoForm onAddTodo={addTodo} />
-      <p>{addTodo}</p>
+      <p>{TodoList}</p>
```

Comment:
you don't need TodoList here too, you already render your list on line 58  <TodoList onRemoveTodo={removeTodo} todoList={todoList} />

Reply:
it is time to update handleAddTodo function inside TodoList component:
it follows ```{title: todoTitle, id:Date.now()}``` object structure when adds a new todo, while you todoList that you reciave from Airtable follows another object structure: 
```{
createdTime: "2023-03-14T00:18:54.000Z"
fields: {Title: 'Grocery shopping '}
id: "recVnH9uEqK7ifOYs"
}```